### ISSUE TITLE
Opportunity categories seem incorrect

### ISSUE THREAD

**Issue by tyler1218hatch** [Riverscapes/riverscapes-tools#102]
![image](https://user-images.githubusercontent.com/27084463/97639341-5bfa6b80-1a0c-11eb-9650-fb7d39535c9f.png)
Most hydro region 17 watersheds looked like this. A tiny amount of 'Easiest - Low-hanging Fruit', no Straight Forward or Strategic, and the vast majority as NA. I'm not sure what's going on here, but it seems like an issue with the logic being used to bin these. Example from 17090003.

**Comment by philipbaileynar** [comment]
I suspect that other issues that you have reported with the conservation module - particularly the land use attributes - are causing something weird. I will investigate.

**Comment by shelbysawyer** [comment]
@philipbaileynar Has there been any progress on getting to the bottom of this?

**Comment by philipbaileynar** [comment]
@tyler1218hatch 17090003 is a massive HUC. Can you suggest the smallest one possible that exhibits this problem?

**Comment by tyler1218hatch** [comment]
16040102 is pretty small, and exhibits this.

**Comment by philipbaileynar** [comment]
@wally-mac and @tyler1218hatch here is some old pyBRAT3 code for conservation and restoration opportunities. Can you confirm if the second occurrence of the greater than sign on line 180 is correct or whether it is meant to be the word "and"?

https://github.com/Riverscapes/pyBRAT/blob/master/Constraints_Opportunities.py#L164-L185

**Existing Code on line 180**

```python
elif occ_hpe >= 5 > occ_ex > 0 and ipc_vlow_lu > 75 and ipc_high_lu < 10:
```

**Should it really be this?**

```python
elif occ_hpe >= 5 and occ_ex > 0 and ipc_vlow_lu > 75 and ipc_high_lu < 10:
```

**Comment by philipbaileynar** [comment]
@wally-mac and @tyler1218hatch I am struggling with the block of pyBRAT3 code above. The code doesn't match the comments. Can we meet to discuss please?

**Comment by tyler1218hatch** [comment]
@philipbaileynar I'm free whenever (except 1-2PM MST tomorrow). I'm not familiar with these classifications, but I think you're right that some of that old code is a bit off.

**Comment by tyler1218hatch** [comment]
@philipbaileynar Looking at the blame for this code, it looks like this was a typo I made 16 months ago. This can and should be easily fixed (also, the old version did have an 'and' there).

**Comment by philipbaileynar** [comment]
Thanks @tyler1218hatch switching the `>` to `and` was already implemented in sqlBRAT. I was just checking.

Now I'm struggling with the comment on line 176 saying "Rare or Occasional", but the `occ_ex > 0` logic on 180 includes frequent and pervasive too. Which is correct @wally-mac ?

**Comment by philipbaileynar** [comment]
Separate question, should the following columns always sum to 100%?

```
iPCVLowLU
iPC_LowLU
iPC_ModLU
iPC_HighLU
```

**Comment by philipbaileynar** [comment]
OK, the questions above notwithstanding, I found an error in the vegetation intensity calculations. This was double counting the areas of vegetation cells in the 100m buffer which in turn was suppressing the vegetation intensity values. Once I fixed this, the iPC VLow, Mod, High LU fields now sum to 100. And this also triggers other clauses in the opportunities code. Here's the report table for 16040102 after the fix:

<img width="629" alt="Screen Shot 2020-11-18 at 4 34 09 PM" src="https://user-images.githubusercontent.com/8645379/99605306-46011a80-29bc-11eb-8216-0e3b8b00e38a.png">

I will publish this fix and make a dataset available in the warehouse for USU to review these columns and then close this issue if they are satistified.
